README.md

```diff
@@ -1,9 +1,19 @@
-# nazareth-recipes
-You know what this is for.
+# :woman_cook::church: Nazareth Recipes 🥘 :man_cook:
 
-## Breakfast
+![Some happy diners enjoying a meal at the Nazareth Guesthouse](https://user-images.githubusercontent.com/23310908/32459347-440e652e-c327-11e7-87f5-0fb24f888b73.jpg)
```

Comment:
I would maybe change this pic? I don't have any but I'm sure someone does! Maybe @Suuuuha?